tests/integration/trace/test_zipkin_trace.py

```diff
@@ -107,3 +125,27 @@ def test_zipkin_trace(trace_server):
             span_id = trace[0][u'span_id']
 
     assert parent_span_id == span_id
+
+
+@pytest.mark.gen_test
+def test_tcollector_submit(monkeypatch, trace_server):
+
+    tchannel = TChannel(name='test')
+
+    hostport = 'localhost:%d' % trace_server.port
```

Comment:
FYI MockServer now provides a hostport attribute. You can just do `trace_server.hostport` instead.